Code/GraphMol/catch_graphmol.cpp

```diff
@@ -2967,6 +2968,34 @@ TEST_CASE("molecules with single bond to metal atom use dative instead") {
   }
 }
 
+TEST_CASE(
```

Comment:
Please add another canonicalization test which verifies that the Cl atoms are also picked in canonical order. Here's a pair of SMILES:
```
F[Pt]1(F)[35Cl][Pt]([Cl]1)(F)Br
F[Pt]1(F)[Cl][Pt]([35Cl]1)(F)Br
```

Reply:
I've done the 100-fold random shuffle I used for the other molecule.